### ISSUE TITLE
Prevent execution to stop when pair that doesn't exist encountered

### ISSUE THREAD

**Issue by iVoider** [fuzzland/ityfuzz#100]
This happens with lots of contracts on all chains like: 0xFFE811714ab35360b67eE195acE7C10D93f89D8C (BSC).
`thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("missing field `data`", line: 1, column: 234)', /ityfuzz/src/evm/onchain/endpoints.rs:734:69`

I had tried to use latest blocks avaliable on api, but problem remains.

Can we just omit such pair like this?
https://github.com/iVoider/ityfuzz/tree/pair-fix

**Comment by iVoider** [comment]
Wrapping is not enough, I guess pair oracle should be turned off in case if pair list is empty.
Otherwise pop from empty list happens here in some contracts:
endpoints.rs 948:
```
let mut peg_info = self
            .get_pair_pegged(token, network, block)
            .get(0)
            .unwrap()
            .clone();
```

**Comment by rappie** [comment]
Are you sure this isn't just this problem?:

https://github.com/fuzzland/ityfuzz/issues/64#issuecomment-1593143022

**Comment by iVoider** [comment]
> Are you sure this isn't just this problem?:
> 
> [#64 (comment)](https://github.com/fuzzland/ityfuzz/issues/64#issuecomment-1593143022)

No, this occurs even on blocks synced by graphapi. And my little fix helps in 90% such cases, as some pairs exists for such contracts.

**Comment by ByteSecurity** [comment]
Hey bro, give this PR a try and see if it can solve the problem.#110

**Comment by iVoider** [comment]
> Hey bro, give this PR a try and see if it can solve the problem.#110

Nope. See another example, when it fails with original missing data exception.
`/ityfuzz/cli/target/release/cli -c BSC --onchain --onchain-block-number 29693964 -f -i --selfdestruct-oracle -p --run-forever --target 0x9e9b768174ef24233bf8ac2f4131f10ff5e72dee`

**Comment by ByteSecurity** [comment]
[fix #110](https://github.com/fuzzland/ityfuzz/pull/110/commits/a236757e4cd231e9413019d028127eca6ca96e98)

The reason for this error is that graphapi returned a value with "null", which caused serde_json to be unable to parse it.

The same question arises in
[invalid type: null](https://github.com/rust-lang/rust-analyzer/issues/979)
[Don't show null deps](https://github.com/rust-lang/cargo/pull/6534)

Thanks a lot
